rmgpy/molecule/pathfinder.py

```diff
@@ -292,13 +292,15 @@ def find_lone_pair_multiple_bond_paths(atom1):
 
     paths = []
     for atom2, bond12 in atom1.edges.items():
-        # Bond must be capable of gaining an order
-        if bond12.is_single() or bond12.is_double():
-            for atom3, bond23 in atom2.edges.items():
-                # Bond must be capable of losing an order without breaking, atom3 must be able to gain a lone pair
-                if atom1 is not atom3 and (bond23.is_double() or bond23.is_triple()) \
-                        and (atom3.is_carbon() or is_atom_able_to_gain_lone_pair(atom3)):
-                    paths.append([atom1, atom2, atom3, bond12, bond23])
+        #if both atom1 and atom2 are sulfur then don't do this type of resonance
+        if not atom1.is_sulfur() or not atom2.is_sulfur():
+           # Bond must be capable of gaining an order
+           if bond12.is_single() or bond12.is_double():
+               for atom3, bond23 in atom2.edges.items():
+                   # Bond must be capable of losing an order without breaking, atom3 must be able to gain a lone pair
+                   if atom1 is not atom3 and (bond23.is_double() or bond23.is_triple()) \
+                           and (atom3.is_carbon() or is_atom_able_to_gain_lone_pair(atom3)):
+                       paths.append([atom1, atom2, atom3, bond12, bond23])
```

Comment:
I think you can combine the top two if statements:
```
if (not atom1.is_sulfur() or not atom2.is_sulfur()) and (bond12.is_single() or bond12.is_double()):
```

Also line 297 is indented by 3 spaces instead of 4.